components/listing/Listing.tsx

```diff
@@ -22,17 +23,17 @@ const Listing: React.FC<ListingProps> = ({ tasks = [] }) => {
       <div className="mt-6 grid grid-cols-1 row-gap-8 bg-orange-600">
         <div
           id="projects-listing-section"
-          className="border-t border-gray-200 pt-8 grid grid-cols-1 row-gap-6 lg:grid-cols-4 lg:gap-5"
+          className="border-t border-gray-200 pt-8 grid grid-cols-1 row-gap-6 lg:grid-cols-3md lg:gap-5 xl:grid-cols-4"
         >
-          <div>
+          <div className="xs: text-center sm:text-left">
             <h3 className="text-lg leading-7 font-medium tracking-tight text-gray-900">Filtry</h3>
             <h3 className="text-lg leading-7 font-medium tracking-tight text-gray-900">
               Sortowanie
             </h3>
           </div>
-          <div className="grid grid-cols-1 row-gap-8 sm:grid-cols-2 sm:col-gap-5 sm:row-gap-6 md:grid-cols-3 lg:col-span-3">
+          <div className="grid grid-cols-1 row-gap-8 pb-8 col-gap-8 sm:grid-cols-2 sm:col-gap-5 sm:row-gap-6 xl:grid-cols-3 xl:col-span-3 mx-4 mb-8">
```

Comment:
do osobnego komponentu:
`/listing/Cards`

Reply:
Wydzieliłem tylko, że inaczej nazwałem ten komponent. Dałem `CardsManager`, bo jestem przyzwyczajony do takiego nazewnictwa. Jeśli Cię to razi to zmienię.